sinon/test/TestSinonSpy.py

```diff
@@ -48,6 +49,7 @@ class TestSinonSpy(unittest.TestCase):
 
     def setUp(self):
         sinon.g = sinon.init(globals())
+        SpyCall.next_spy_call_id = 0
```

Comment:
If the code works well, we have two options here.

1. Put this initialization into [sandbox.py](https://github.com/note35/sinon/blob/master/sinon/lib/sandbox.py), making it cleanup every time.

2. Don't care.

Reply:
Hi Kir,

The code `SpyCall.next_spy_call_id = 0` is not there to prevent an integer overflow. It's there to prevent cross-contamination between tests, so that each SpyCall starts with an ID of 0.

Best Regards,

--Jonathan

Reply:
I understand your intention here, what I want to say is, if there is "cross-contamination between tests", you need to put `SpyCall.next_spy_call_id = 0` into sandbox, instead of making the library user do this by themselves.

Reply:
Oh... I didn't understand what `sandbox.py` was doing. I just noticed now that the `@sinontest` annotation must be executing the `sinontest` function in `sandbox.py`. I'll move `SpyCall.next_spy_call_id = 0` into `sandbox.py`.